types/css-tree/index.d.ts

```diff
@@ -518,7 +518,7 @@ export function generate(ast: CssNode, options?: GenerateOptions): string;
 export interface WalkContext {
     root: CssNode;
     stylesheet: StyleSheet;
-    atrule: Atrule;
+    atrule: Atrule | null;
```

Comment:
Seems to me that all those members can be null, except for root. In that case, we should change all of them

Reply:
Fixed, thanks for the suggestion!